### DISCUSSION TITLE
The name of the variable as a title in it's histogram

### DISCUSSION THREAD

**Discussion by NikitaSakharov** [mtex-toolbox/mtex#989] [Ask Anything] [answered]
Hi everyone!

Cuold you please help me?

How can I get the name of my grains variable (let us say it's called mergedGrains)?
I just wanna edit the code of 'histogram' function in such a way, that I could place the name of the variable in the legend box.

thanks in advance!

Sincerely, Nikita

**Comment by ralfHielscher** [comment, marked as answer]
Hi Nikita,

see here: https://www.mathworks.com/help/matlab/ref/inputname.html

Ralf.

**Reply by NikitaSakharov** [reply]
Thanks, Ralf!

I'll look at this.
